.changeset/eighty-taxis-repair.md

```diff
@@ -0,0 +1,5 @@
+---
+'@matrix-widget-toolkit/testing': major
```

Comment:
We are dropping jest support. So this is a major difference, as it breaks existing consumers entirely if they rely on jest. An alternative would be to make a new package for vitest and deprecate this package. (which still imho is a major release?)

Reply:
Fine with that